flash/vision/backbones.py

```diff
@@ -32,12 +33,36 @@
 BOLTS_MODELS = ["simclr-imagenet", "swav-imagenet"]
 
 
-def backbone_and_num_features(model_name: str, *args, **kwargs) -> Tuple[nn.Module, int]:
+def backbone_and_num_features(
+    model_name: str,
```

Comment:
Can you add doctoring with args description ?

Reply:
Added!